src/test/scala/it/bitrock/dvs/streams/topologies/FlightInterpolatedListStreamSpec.scala

```diff
@@ -94,7 +94,7 @@ class FlightInterpolatedListStreamSpec extends Suite with AnyWordSpecLike with E
     }
   }
 
-  private def compareFlights(interpolated: FlightReceived, original: FlightReceived): Assertion = {
+  private def compareFlights(interpolated: FlightInterpolated, original: FlightReceived): Assertion = {
```

Comment:
Can we add a test for updated and interpolated fields?
Updated should be the same as original flight and interpolated should be greater than that but lower than now.

What do you think?

Reply:
you mean assertions on the test? I agree.
I was thinking about adding a whole new test too, but maybe does not make sense.

Reply:
I'm fine with your implementation, I meant assertions